# StrArrKV.msg
string key
string[] values